source/css/db-ui-core.vars.scss

```diff
@@ -4,10 +4,8 @@ $images-path: "../images/" !default;
 $fonts-path: "../fonts/" !default;
 
 // General variables by DB UI Base
-@import "@db-ui/base/build/css/variables";
+@import "@db-ui/base/build/scss/db-ui-base";
```

Comment:
this might be misleading, as `source/css/db-ui-core.vars.scss` might get interpreted to only include variables, but `@db-ui/base/build/scss/db-ui-base` even also includes styling declarations. Let's discuss, probably we need some further clustering for the `@db-ui/base/build/scss/db-ui-base` contents.